joint_position_controller:

  type: joint_position_controller/JointPositionController

  robot_description: /robot_description

  limit_error: false

  q_err_max: 0.01
  q_err_dot_max_factor: 2.0

  PGain: 500.0
  DGain: 200000.0
  DGainFactor: 0.7
  
  Kp_0:  0.5
  Kp_1:  0.5
  Kp_2:  0.5
  Kp_3:  0.5
  Kp_4:  0.5
  Kp_5:  0.5
  Kp_6:  0.5
  Kp_7:  0.5
  Kp_8:  0.5
  Kp_9:  0.5
  Kp_10: 0.5
  Kp_11: 0.5
  Kp_12: 0.5
  Kp_13: 0.6
  Kp_14: 0.6
  Kp_15: 1.0
  Kp_16: 1.0
  Kp_17: 1.0
  Kp_18: 1.0
  Kp_19: 1.0
  Kp_20: 1.0
  Kp_21: 1.0
  Kp_22: 1.0  
  Kp_23: 0.5
  Kp_24: 0.5
  Kp_25: 0.5
  Kp_26: 0.5
  Kp_27: 0.5
  Kp_28: 1.0
  Kp_29: 1.0
  Kp_30: 1.0
  Kp_31: 1.0
  Kp_32: 1.0
  Kp_33: 1.0
  Kp_34: 1.0  
  Kp_35: 1.0  
  Kp_36: 0.5  
  Kp_37: 0.5        

  Kd_0:  0.5
  Kd_1:  0.5
  Kd_2:  0.5
  Kd_3:  0.5
  Kd_4:  1.5
  Kd_5:  1.5
  Kd_6:  1.0
  Kd_7:  1.0
  Kd_8:  0.9
  Kd_9:  0.9
  Kd_10: 0.6
  Kd_11: 0.6
  Kd_12: 0.6
  Kd_13: 1.0
  Kd_14: 1.0
  Kd_15: 0.9
  Kd_16: 0.9
  Kd_17: 0.6
  Kd_18: 0.6
  Kd_19: 0.6
  Kd_20: 0.6
  Kd_21: 0.6
  
  tau_max_value: 40.0
  tau_max_factor_0:  0.0
  tau_max_factor_1:  0.0
  tau_max_factor_2:  0.0
  tau_max_factor_3:  0.0
  tau_max_factor_4:  1.5
  tau_max_factor_5:  1.5
  tau_max_factor_6:  1.0
  tau_max_factor_7:  1.0
  tau_max_factor_8:  0.9
  tau_max_factor_9:  0.9
  tau_max_factor_10: 0.6
  tau_max_factor_11: 0.6
  tau_max_factor_12: 0.6
  tau_max_factor_13: 1.0
  tau_max_factor_14: 1.0
  tau_max_factor_15: 0.9
  tau_max_factor_16: 0.9
  tau_max_factor_17: 0.6
  tau_max_factor_18: 0.6
  tau_max_factor_19: 0.6
  tau_max_factor_20: 0.6
  tau_max_factor_21: 0.6
  tau_max_factor_22: 0.5
  tau_max_factor_23: 0.5
  
